source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,597 @@
+
+~~~~~~
+
+* finally, the user is required to update the ports, indicating which security
+groups apply to it. The following playbook shows how a single port, referenced
+by name, is updated:
+
+~~~~~~
+
+- os_port:
+    cloud: ovirt
+    state: present
+    name: <port_name>
+    security_groups:
+        - "{{ my_app_default_security_group.id }}"
+
+~~~~~~
+
+### Installation/Upgrade
+
+The install time required updates are described in the section [above](#engine-setup).
+
+On upgrades with existing VMs - having attachments to external networks -
+the ports will **not** be updated, and it will be up to the administrator to
+manually activate port security in the existent ports.
+
+Updating the network's port-security-enabled attribute **will not** cascade to
+the existent ports, as defined [in the Networking API](https://developer.openstack.org/api-ref/network/v2/#port-security).
+
+### Engine-Setup
+
+A new ovirt-provider-ovn related question will be added, asking the user if
+port-security is enabled at network level. The default answer will be 'yes'.
+
+This will lead to all the created ports having port-security active, leading to
+all IP traffic being dropped.
+
+## Design feature descrition
+
+### Mandatory security group creation
+
+There are 2 types of required security groups, that are created automatically
+by the ovirt-provider-ovn, when required. They are:
+
+1. the **deny-all** security group, provisioned when the first port with the
+**port-security** flag activated is created. This port will be the container for
+the rules that drop all IP traffic. More information can be found in
+[Activating the feature](#activating-the-feature).
+
+
+2. the **allow-dhcp** security group. Creating a subnet on top of an external
+network will provision an **allow-dhcp** security group **for that particular
+network**. Refer to the [creating a subnet](#creating-a-subnet-on-top-of-a-network) section for more information.
+
+The ansible representation of these groups is:
+~~~~~
+- os_security_group:
+    cloud: ovirt
+    state: present
+    name: deny-all
+    description: security group to drop all IP traffic
+
+- os_security_group:
+    cloud: ovirt
+    state: present
+    name: allow-dhcp-<subnet_uuid>
+    description: security group to allow dhcp-traffic
+~~~~~
+
+***NOTE:*** the **deny-all** security group can be provisioned through ansible,
+during the engine-setup stage. the **allow-dhcp** groups will be created as
+part of the subnet creation workflow.
+
+### Initial rule creation
+
+Since the Openstack Networking API is a white list, the **deny-all** related 
+rules cannot be triggered from the API itself.
+
+To avoid the need to create the rules beforehand, these ACLs will be
+provisioned when the first port having the *port-security* attribute enabled is
+created.
+
+The DHCP related rules will be provisioned after the respective group is
+created, and as part os the [create subnet](#creating-a-subnet-on-top-of-a-network)
```

Comment:
Please change "part os.." to "part of.."

Reply:
Done